### DISCUSSION TITLE
Call for images/volumes historically used with ASPIRE

### DISCUSSION THREAD

**Discussion by garrettwrong** [ComputationalCryoEM/ASPIRE-Python#929] [General]
Starting this thread as a place to write down any historical images or volumes used with ASPIRE.  Items that were used in previous projects, or graphics in papers, presentations, classes etc.  There is already an effort to keep track of datasets (empiar/emdb/etc), so this is more focused on one offs designed for demostration/testing.

We are working on a background project to centralize these items and provide a simple interface to download such things.  Ultimately the goal is to be able to reproduce useful representations.  As an example, we've reconstructed a dice image for the CTF tutorial from lecture notes.

If we're browsing a paper/code and see a useful image, please jot it down with a reference.

Example ones that I am aware of:

The (partially rotated) volume used in the weighted/spectral volume project (https://arxiv.org/pdf/1907.01898.pdf).
There is a clock associated with the same paper ^ as well.
The hedgehog, (https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8813033/pdf/nihms-1739531.pdf)

**Comment by j-c-c** [comment]
I came across [this file](https://github.com/PrincetonUniversity/aspire/tree/e697eee5bb185ab1962f1d6e5becaf9caa277732/projections/simulation/models) that contains parameters to generate some legacy volumes using [this gaussian phantom volume method](https://github.com/PrincetonUniversity/aspire/blob/e697eee5bb185ab1962f1d6e5becaf9caa277732/projections/simulation/cryo_gaussian_phantom_3d.m).
